Modules/_ctypes/_ctypes.c

```diff
@@ -4054,6 +4059,8 @@ _init_pos_args(PyObject *self, PyTypeObject *type,
     }
 
     dict = PyType_stgdict((PyObject *)type);
+    if (dict == NULL)
```

Comment:
Aaaaah, thanks for the confirmation @brettcannon ! I had a long discussion with @serhiy-storchaka about that ;-)